src/components/toolbox/copyToClipboard/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { CopyToClipboard as ReactCopyToClipboard } from 'react-copy-to-clipboard';
+import { translate } from 'react-i18next';
+import Icon from '../icon';
+import styles from './copyToClipboard.css';
+import Piwik from '../../../utils/piwik';
+
+class CopyToClipboard extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      copied: false,
+    };
+  }
+
+  textIsCopied() {
+    Piwik.trackingEvent('CopyToClipboard', 'button', 'Copy');
+    this.setState({
+      copied: true,
+    });
+    setTimeout(() => {
+      this.setState({
+        copied: false,
+      });
+    }, 3000);
+  }
+
+  render() {
+    const {
+      value, t, className, text, copyClassName,
+    } = this.props;
+    return (
+      <div onClick={(e) => {
+        e.stopPropagation();
+      }}>
+        <ReactCopyToClipboard text={value} onCopy={() => this.textIsCopied()}>
```

Comment:
bind the function on the constructor, to keep it consistent with other components.